### ISSUE TITLE
Not working in beta 1.12.5

### ISSUE THREAD

**Issue by Eldar-X** [jrattray13/swackets#24]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.12.0-beta5
**Electron Version**: 1.3.6
**System**: Mac OS X 10.12.1
**Thrown From**: [swackets](https://github.com/jrattray13/swackets) package, v0.12.0
### Stack Trace

Failed to activate the swackets package

```
At Invalid regular expression: /(?<!\\)(?:[$%#].?)?[\{\[]/: Invalid group

SyntaxError: Invalid regular expression: /(?<!\\)(?:[$%#].?)?[\{\[]/: Invalid group
    at RegExp (native)
    at SwacketsView.module.exports.SwacketsView.config (/Users/elix/.atom/packages/swackets/lib/swackets-view.coffee:64:26)
    at new SwacketsView (/Users/elix/.atom/packages/swackets/lib/swackets-view.coffee:13:19)
    at Object.activate (/Users/elix/.atom/packages/swackets/lib/swackets.coffee:32:21)
    at Package.module.exports.Package.activateNow (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package.js:184:19)
    at /Applications/Atom Beta.app/Contents/Resources/app.asar/src/package.js:157:32
    at Package.module.exports.Package.measure (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom Beta.app/Contents/Resources/app.asar/src/package.js:150:26
    at Package.module.exports.Package.activate (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package.js:147:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package-manager.js:548:21)
    at /Applications/Atom Beta.app/Contents/Resources/app.asar/src/package-manager.js:529:29
    at Config.module.exports.Config.transactAsync (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package-manager.js:524:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/package-manager.js:506:46)
    at /Applications/Atom Beta.app/Contents/Resources/app.asar/src/atom-environment.js:783:28
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "intellij-idea-keymap",
      "tool-bar-atom",
      "atom-autocomplete-php"
    ],
    "telemetryConsent": "no",
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  },
  "swackets": {
    "colorSquareBrackets": true
  }
}
```
### Installed Packages

``` coffee
# User
activate-power-mode, v1.1.0 (inactive)
atom-autocomplete-php, v0.19.4 (inactive)
atom-beautify, v0.29.13 (inactive)
atom-material-syntax, v0.4.6 (inactive)
atom-material-ui, v1.3.6 (inactive)
atom-ternjs, v0.15.0 (inactive)
atom-wrap-in-tag, v0.6.0 (inactive)
auto-detect-indentation, v1.3.0 (inactive)
autoclose-html, v0.23.0 (inactive)
autocomplete-json, v5.1.0 (inactive)
autocomplete-modules, v1.6.7 (inactive)
color-picker, v2.2.2 (inactive)
docblockr, v0.8.7 (inactive)
double-tag, v0.4.0 (inactive)
emmet, v2.4.3 (inactive)
file-icons, v1.7.23 (inactive)
filecolor, v0.4.0 (inactive)
git-plus, v5.18.0 (inactive)
grammar-token-limit, v0.1.1 (inactive)
highlight-selected, v0.11.2 (inactive)
hyperclick, v0.0.39 (inactive)
intellij-idea-keymap, v0.2.3 (inactive)
language-marko, v2.0.10 (inactive)
language-pug, v0.0.19 (inactive)
linter, v1.11.18 (inactive)
linter-eslint, v8.0.0 (inactive)
material-ui, v1.0.9 (inactive)
minimap, v4.25.6 (inactive)
minimap-highlight-selected, v4.4.0 (inactive)
native-ui, v0.19.1 (inactive)
node-debugger, v1.9.1 (inactive)
open-terminal-here, v2.1.4 (inactive)
pigments, v0.37.0 (inactive)
project-manager, v3.3.2 (inactive)
smart-tags, v0.1.5 (inactive)
svg-preview, v0.10.0 (inactive)
swackets, v0.12.0 (inactive)
todo-show, v1.7.0 (inactive)
tool-bar, v1.0.1 (inactive)
tool-bar-almighty, v0.8.1 (inactive)
tool-bar-atom, v1.7.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (inactive)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (inactive)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (inactive)
archive-view, v0.62.0 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.13.1 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.4 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.42.0 (inactive)
bracket-matcher, v0.82.2 (inactive)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.202.0 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.60.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (inactive)
metrics, v1.0.0 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (inactive)
snippets, v1.0.3 (inactive)
spell-check, v0.68.4 (inactive)
status-bar, v1.4.1 (inactive)
styleguide, v0.47.2 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.102.2 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.210.0 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (inactive)
wrap-guide, v0.38.2 (inactive)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.0 (inactive)
language-csharp, v0.12.1 (inactive)
language-css, v0.40.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.43.0 (inactive)
language-html, v0.46.1 (inactive)
language-hyperlink, v0.16.1 (inactive)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (inactive)
language-json, v0.18.3 (inactive)
language-less, v0.29.6 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (inactive)
language-php, v0.37.3 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.1 (inactive)
language-ruby, v0.70.2 (inactive)
language-ruby-on-rails, v0.25.1 (inactive)
language-sass, v0.57.0 (inactive)
language-shellscript, v0.23.0 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.25.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.29.1 (inactive)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (inactive)
language-yaml, v0.27.1 (inactive)

# Dev
No dev packages
```

**Comment by skamansam** [comment]
I am getting the same error, on Fedora 24 Linux.

**Comment by ryanmagoon** [comment]
+1, MacOS 10.12.1

**Comment by grma1** [comment]
+1, Atom: 1.12.0 release build, swackets: 0.12.0

**Comment by dario-zubovic** [comment]
+1, Atom 1.12.0 on Windows 10, swackets build 0.12.0

**Comment by maxpower9000** [comment]
+1, Atom 1.12.1 on Win 7 Pro, swackets 0.12.0

**Comment by palfrey** [comment]
This looks like it was caused by #23, specifically the `notEscapedModifier`. Commented out that bit and the problem went away locally.

**Comment by palfrey** [comment]
AFAIK negative lookbehinds aren't supported by Javascript at all. http://stackoverflow.com/questions/641407/javascript-negative-lookbehind-equivalent suggests some workarounds but not sure if any of those will work here. Any thoughts @dkniffin?

**Comment by dkniffin** [comment]
Hmm. Odd. If it's not supported in javascript at all, I'm curious why this works on my machine. I will definitely look into it.

**Comment by dkniffin** [comment]
I've got a fix up at #28, which removes those regexes.
